src/unittests/maths.spec.ts

```diff
@@ -210,68 +273,68 @@ g.test('test,math,correctlyRounded')
     // Zeros
     { test_val: f32Bits(kBit.f32.positive.zero), target: 0, is_correct: true },
     { test_val: f32Bits(kBit.f32.negative.zero), target: 0, is_correct: true },
-    { test_val: f32Bits(kBit.f32.subnormal.positive.min), target: 0, is_correct: false },
-    { test_val: f32Bits(kBit.f32.subnormal.positive.max), target: 0, is_correct: false },
-    { test_val: f32Bits(kBit.f32.subnormal.negative.max), target: 0, is_correct: false },
-    { test_val: f32Bits(kBit.f32.subnormal.negative.min), target: 0, is_correct: false },
+    { test_val: f32Bits(kBit.f32.subnormal.positive.min), target: 0, is_correct: true },
```

Comment:
It's possible that there might be tests we'll need to write that'll want to use `correctlyRounded` without thet FTZ behaviour.
For this reason, I'd prefer it if exposed this as a parameter flag (which can happily default to testing with and without FTZ), but make sure that we have tests for both behaviours. I think you can just copy the parent-change test's as another `g.test('test,math,correctlyRoundedNoFlushToZero)` set.

Reply:
I have added control flags and additional testing